config/default/manager_auth_proxy_patch.yaml

```diff
@@ -20,6 +20,13 @@ spec:
         - containerPort: 8443
           protocol: TCP
           name: https
+        resources:
+          limits:
+            cpu: 200m
+            memory: 128Mi
+          requests:
```

Comment:
Could we set the limits of our resources as QOS guaranteed?

https://github.com/opdev/l5-operator-demo/runs/6309421552?check_suite_focus=true

Reply:
don't think its needed the current allocation may already be too much i think..